src/cloud-api-adaptor/docs/addnewprovider.md

```diff
@@ -53,10 +53,366 @@ To include your provider you need reference it from the main package. Go build t
 ```
 Note the comment at the top of the file, when building ensure `-tags=` is set to include your new provider. See the [Makefile](../../cloud-api-adaptor/Makefile#L26) for more context and usage.
 
-#### Step 3: Add documentation on how to build a Pod VM image
+### Step 3: Add documentation on how to build a Pod VM image
 
 For using the provider, a pod VM image needs to be created in order to create the peer pod instances. Add the instructions for building the peer pod VM image at the root directory similar to the other providers.
 
-#### Step 4: Add E2E tests for the new provider
+### Step 4: Add E2E tests for the new provider
 
 For more information, please refer to the section on [adding support for a new cloud provider](../test/e2e/README.md#adding-support-for-a-new-cloud-provider) in the E2E testing documentation.
+
+
+# :memo: Adding support for a new provider (Runtime-time plugin)
+Follow this section user just need build the new cloud provider "example" plugin file, and modify "peer-pods-cm" configmap and "cloud-api-adaptor" damonset, do NOT need re-build `cloud-api-adaptor` and `peerpod-ctrl` docker images.
+Assume you are using the `cloud-api-adaptor` and `peerpod-ctrl` image with the Runtime-time plugin function support.
+
+And the cloud-api-adaptor source code at `/root/cloud-api-adaptor` on your build machine.
```

Comment:
This could be any path right (for eg `$HOME/cloud-api-adaptor`) and not necessarily `/root/cloud-api-adaptor` ?

Reply:
yeah, it can be any path, use in my dev build machine I use `/root/cloud-api-adaptor`.
Will update to use `$HOME/cloud-api-adaptor`

Reply:
updated.